src/arithmetic.c

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file arithmetic.c
+ * @author David Chung
+ * @brief Function prototyping of all arithmetic functions to be used in the main program.
+ * @version 2.1
+ * @date 2024-11-25
+ * 
+ */
+
+#include "arithmetic.h"
+#include "debug.h"
+
+/**
+ * @brief Addition function.
```

Comment:
Doxygen comments usually go in header (.h) files unless they are static functions in which case they would be in the .c files. But this looks good!